pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -64,10 +64,8 @@ func TestPutOrPatchOpenShiftClusterAdminAPI(t *testing.T) {
 				f.AddSubscriptionDocuments(&api.SubscriptionDocument{
 					ID: mockSubID,
 					Subscription: &api.Subscription{
-						State: api.SubscriptionStateRegistered,
-						Properties: &api.SubscriptionProperties{
-							TenantID: "11111111-1111-1111-1111-111111111111",
-						},
+						State:      api.SubscriptionStateRegistered,
+						Properties: &api.SubscriptionProperties{},
```

Comment:
is this change simply because the TenantID wasn't being used in the test?  If so, out of interest, can the `Properties: &api.SubscriptionProperties{},` be removed too or is that needed?